Firestore/Protos/protos/google/firestore/v1/common.proto

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package google.firestore.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+option csharp_namespace = "Google.Cloud.Firestore.V1Beta1";
```

Comment:
I'm taking the protos as generated using the scrubber in google3. That was just missed there. I'll follow up there and re-export.

Note that these are the C# and php namespaces so it doesn't really affect what we're doing. These do show up in the generated files, but only because the text form of the proto (including its options) is compiled into a descriptor you can use for reflection at run-time. We don't use those features so this has no practical consequence.